tools/autograd/derivatives.yaml

```diff
@@ -221,6 +221,10 @@
 - name: fill_(Tensor self, Scalar value)
   self: zeros_like(grad)
 
+- name: fill_(Tensor self, Tensor value)
```

Comment:
How can you fill with a tensor?

Reply:
It has to be zero dimensional; we added the overload so things like t.fill_(v.sum()) would continue to work now that sum() returns a 0-dimensional tensor (to match desired autograd semantics).